src/main/scala/is/hail/methods/LogisticRegressionBurden.scala

```diff
@@ -0,0 +1,97 @@
+package is.hail.methods
+
+import breeze.linalg._
+import is.hail.expr._
+import is.hail.keytable.KeyTable
+import is.hail.stats._
+import is.hail.utils._
+import is.hail.variant._
+import org.apache.spark.sql.Row
+
+object LogisticRegressionBurden {
+
+  def apply(vds: VariantDataset,
+    keyName: String,
+    variantKeys: String,
+    singleKey: Boolean,
+    aggExpr: String,
+    test: String,
+    yExpr: String,
+    covExpr: Array[String]): (KeyTable, KeyTable) = {
+
+    def tests = Map("wald" -> WaldTest, "lrt" -> LikelihoodRatioTest, "score" -> ScoreTest, "firth" -> FirthTest)
```

Comment:
This is repeated from above and is still not a `def` nor should it be local to this method. 

`LogisticRegressionTest` should have a companion object that has a map of all the tests in one place.